azurerm/resource_arm_netapp_account_test.go

```diff
@@ -0,0 +1,190 @@
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAzureRMNetAppAccount_update(t *testing.T) {
+	resourceName := "azurerm_netapp_account.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMNetAppAccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMNetAppAccount_basic(ri, location),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMNetAppAccountExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "active_directory.#", "0"),
+				),
+			},
+			{
+				Config: testAccAzureRMNetAppAccount_complete(ri, location),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMNetAppAccountExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "active_directory.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "1"),
+					resource.TestCheckResourceAttr(resourceName, "tags.ENV", "Test"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testCheckAzureRMNetAppAccountExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("NetApp Account not found: %s", resourceName)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		client := testAccProvider.Meta().(*ArmClient).Netapp.AccountClient
+		ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+		if resp, err := client.Get(ctx, resourceGroup, name); err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: NetApp Account %q (Resource Group %q) does not exist", name, resourceGroup)
+			}
+			return fmt.Errorf("Bad: Get on netapp.AccountClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMNetAppAccountDestroy(s *terraform.State) error {
+	client := testAccProvider.Meta().(*ArmClient).Netapp.AccountClient
+	ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_netapp_account" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		if resp, err := client.Get(ctx, resourceGroup, name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Get on netapp.AccountClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMNetAppAccount_basic(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
trying something new, could we use 
```suggestion
  name     = "acctestRG-netapp-%d"
```

Reply:
Fixed